homeassistant/components/light/__init__.py

```diff
@@ -111,7 +110,7 @@
     ATTR_EFFECT: cv.string,
 })
 
-LIGHT_TURN_OFF_SCHEMA = vol.Schema({
+LIGHT_TURN_OFF_SCHEMA = ENTITY_SERVICE_SCHEMA.extend({
     ATTR_ENTITY_ID: cv.comp_entity_ids,
```

Comment:
Entity of should be removed here right? Its part of base schema?

Reply:
Good catch